iceoryx_posh/include/iceoryx_posh/mepoo/chunk_header.hpp

```diff
@@ -25,18 +26,45 @@ namespace iox
 {
 namespace mepoo
 {
+/// @brief Helper struct to use as default template parameter when no custom header is used
+struct NoCustomHeader
```

Comment:
To be more efficient at compile time I suggest the following approach:
```cpp
struct NoCustomHeader_t {};
constexpr NoCustomHeader_t NoCustomHeader{};
```

Then you can use it without always creating a new empty struct.

Reply:
the struct will never be instantiated since it's sole purpose is to live the life of a default template parameter